### PR TITLE
removed internet explorer and enhanced controller references

### PR DESCRIPTION
**Pull request by dillonreese**

### CODE REVIEW

**Review by JayneSoliz — CHANGES_REQUESTED** [review]
Dillon, 
Here are some things I noticed...
First section
* The sentence between 355 and 1460 needs a ":" at the end? - @@ -1460,8 +1458,6 @@ The following settings on the advanced tab allow you to customize the client use
* Bullet point at 1467 "Allows" should be capitalized and a "." at the end of the sentence
* At 354, there is a space before the colon, 1. **Require Hyper-V :** Checking this box
* At 38, you can remove "other", - In other browsers such as **Google Chrome** or **Mozilla Firefox**;
* At 106, it's should be its, - **Enable Dynamic Screen Resizing**: Allows the VM to automatically resize it's resolution
* At 119, you're should be your, If a lab uses an iframe integration, 3rd party cookies .... If you're browser is blocking 3rd party cookies, 
* At 13 and 15, there are no periods at the end of these bullet points. 1. Websocket connection upgrade request (ws:// or wss://) headers must not be altered ... 1. Certificates must not be altered or repackaged
* At 1279, Allows should be capitalized and there is no "." at the end of the bullet point. - **Allow User to Cancel Labs:** allows the user to cancel the lab at any point

That's all I saw.

File: tms/connectivity-requires.md [hunk 1]
```diff
@@ -118,8 +118,6 @@ The majority of VM or container based labs can be accessed via HTML5 websocket c
 
 If a lab uses an iframe integration, 3rd party cookies must not be blocked by the web browser used to access the lab. If 3rd party cookies are blocked, an _Access Denied_ message will be displayed when launching the lab. Most web browsers do not block 3rd party cookies by default. If you're browser is blocking 3rd party cookies, please check with the browser's vendor to learn more about how third party cookies may be blocked.
```
Comment (dillonreese) [inline tms/connectivity-requires.md:119]: "you're" should be "your"

Comment (dillonreese) [reply tms/connectivity-requires.md:119]: changed

**Review by JayneSoliz — COMMENTED** [review]
Hey there,
You still need periods at the end of 1463 and 1277 - the first and the last changes.

**Comment by dillonreese** [comment]
Jayne,

Thank you for the review, I have put in the changes you suggested.

In the future, would you mind posting suggested changes in GitHub, using their Comment feature?
That would make it easier to determine which suggested changes belong with which document, instead of searching across the pull request for the suggested text and seeing which file it belongs to.

You can put your cursor in the document where you want to suggest changes and click the blue + icon and that will open a text box that you can put comments into. Example:
***@***.***

**Comment by dillonreese** [comment]
Jayne,

Which file are those changes needed on?

**Comment by dillonreese** [comment]
Resending this because I don’t think it was sent to you. It was sent to some GitHub addresses.

From: Dillon Reese
Sent: Monday, August 29, 2022 11:58 AM
To: LearnOnDemandSystems/docs ***@***.***>; LearnOnDemandSystems/docs ***@***.***>
Cc: Author ***@***.***>
Subject: RE: [LearnOnDemandSystems/docs] removed internet explorer and enhanced controller references (PR #852)

Jayne,

Thank you for the review, I have put in the changes you suggested.

In the future, would you mind posting suggested changes in GitHub, using their Comment feature?
That would make it easier to determine which suggested changes belong with which document, instead of searching across the pull request for the suggested text and seeing which file it belongs to.

You can put your cursor in the document where you want to suggest changes and click the blue + icon and that will open a text box that you can put comments into. Example:
***@***.***

File: guides/idl2/idlv2-authoring-guide-and-best-practice.md [hunk 2]
```diff
@@ -1460,11 +1458,9 @@ The following settings on the advanced tab allow you to customize the client use
 
 - **Show Virtual Machine Power Options:** Checking this box enables virtual machine power options to be available to users.
 
-- **Require Hyper-V Enhanced Controller:** Checking this box requires the user to use the Hyper-V Enhanced controller to work in the lab. If this box is checked, users will not be able to use any other machine remote controllers. Note that the Hyper-V enhanced controller only allows one user to access the lab at a time. 
-
 - **Save/Cancel Options:** 
 
-    - **Allow User to Cancel Labs:** allows the user to cancel the lab at any point
+    - **Allow User to Cancel Labs:** Allows the user to cancel the lab at any point
```
Comment (JayneSoliz) [inline guides/idl2/idlv2-authoring-guide-and-best-practice.md:1463]: Needs a period at the end.

File: tms/token_store.md [hunk 3]
```diff
@@ -1272,11 +1272,9 @@ The following settings on the advanced tab allow you to customize the client use
 
 - **Show Virtual Machine Power Options:** Checking this box enables virtual machine power options to be available to users.
 
-- **Require Hyper-V Enhanced Controller:** Checking this box requires the user to use the Hyper-V Enhanced controller to work in the lab. If this box is checked, users will not be able to use any other machine remote controllers. Note that the Hyper-V enhanced controller only allows one user to access the lab at a time. 
-
 - **Save/Cancel Options:** 
 
-    - **Allow User to Cancel Labs:** allows the user to cancel the lab at any point
+    - **Allow User to Cancel Labs:** Allows the user to cancel the lab at any point
```
Comment (JayneSoliz) [inline tms/token_store.md:1277]: Needs a period at the end.

**Pull request merged**